apps/authoring/src/renderer/components/portal/comp-portal.tsx

```diff
@@ -0,0 +1,20 @@
+import { useEffect, useRef } from 'react';
```

Comment:
I created a portal to display modals and toasts. For reference, I used it in the publish component.